.rubocop.yml

```diff
@@ -0,0 +1,151 @@
+require:
+  - rubocop-performance
+  - rubocop-rspec
+
+inherit_mode:
+  merge:
+    - Exclude
+
+AllCops:
+  TargetRubyVersion: 2.3
```

Comment:
Ruby 2.3 reached EOL and 2.4 will do the same in a few months. Rails 6 supports only ruby >= 2.5.
We about supporting only ruby >= 2.5?

Reply:
The problem is that ruby 2.2 is still used by people that depend on `sidekiq-unique-jobs`. I've been pushing them towards 2.4 so I think that's a better starting point. Keep in mind that when doing open source work it can be a real pain in the neck with the ruby versions. I for one love this:

```ruby
[1, 2, 3, 4].each_with_object({}) do |hash, item|
  # perform some business logic
rescue RuntimeError => ex 
  logger.debug(ex)
end
```

Unfortunately, I still have to hold off on that because people are on old rubies. That is why 2.3 was there but 2.4 I think is fine now. For people on newer sidekiq versions, they will have to use newer ruby so maybe two versions then? I don't mind that.

Reply:
I changed it to ruby 2.5. Since noone is using the gem yet I think that's fine!